workflows/rnaseq/downstream/helpers.Rmd

```diff
@@ -357,6 +358,64 @@ DESeqDataSetFromSalmon <- function (sampleTable, directory='.', design,
     return(object)
 }
 
+#' Make a list of dds objects
+#'
+#' This function takes a list of sample, design pairs, plus other info, to output
+#' a list of DESeq object. 
+#'
+#' @param deseq_obj_list A named list of lists. Each list should have two to three elements:
+#'       1 should be a sampletable, 2 should be a design. The 3rd is an optional list of arguments
+#'       to pass to the DESeqDataSet constructor.
+#' @param salmon.files txi object from tximport if transcript-level counts desired; NULL forces featurecounts; 
+#'        shared by all dds.
+#' @param combine.by The column to collapse technical replicates by.
+#' @param remove.version Boolean to decide whether to remove version information 
+#' @return A list of dds objects.
```

Comment:
could use some more detailed docs here, I'll take a stab at it an push to this PR